cypress/e2e/tests.cy.js

```diff
@@ -0,0 +1,108 @@
+import userResponse from "../fixtures/userResponse.json";
+import ObjectPage from "../pages/objectsPage.cy";
+
+const objectPage = new ObjectPage();
+
+describe("App example - Cypress workshop", () => {
```

Comment:
let's change description to something meaningful and as a real example - Smoke tests or Basic test suite